bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -216,24 +216,95 @@ <h3 class="section-heading">{{ _('What we’re reading:') }}</h3>
 
 {# https://github.com/mozilla/bedrock/issues/5741 #}
```

Comment:
I suggest wrapping this in a block called `{% block structured_data %}`.

You can then override this block in the German and French homepage templates to prevent it being included:

In `home-de.html` and `home-fr.html`:

```
{% block structured_data %}{% endblock %}
```

If you wanted to go a step further, you could also move the data itself into to a separate html include, to keep the homepage template more readable.

In `home-en.html`:

```
{% block structured_data %}
  {% include 'mozorg/home/includes/structured-data.html' %}
{% endblock %}
```

Reply:
Thank you Alex. I will include these changes.